grafana-plugin/integration-tests/utils/forms.ts

```diff
@@ -39,7 +39,7 @@ export const clickButton = async ({
   dataTestId,
 }: ClickButtonArgs): Promise<void> => {
   const baseLocator = dataTestId ? `button[data-testid="${dataTestId}"]` : 'button';
-  const button = (startingLocator || page).locator(`${baseLocator} >> text=${buttonText}`);
+  const button = (startingLocator || page).locator(`${baseLocator}:not([disabled]) >> text=${buttonText}`);
```

Comment:
this _should_ make some of the flaky schedules tests more reliable.. I think the issue was that the tests were trying to click on the "Add rotation" button, which according to this screenshot, starts off `disabled`. By adding this, Playwright _should_ wait until the desired button is enabled before trying to click it:
<img width="253" alt="Screenshot 2023-03-30 at 16 12 25" src="https://user-images.githubusercontent.com/9406895/228864891-413ed12d-6735-419c-9674-524cdce3873d.png">